### PR TITLE
Installed tailwindcss and removed base style

### PR DESCRIPTION
**Pull request by igMike-V**
Added Tailwindcss

### CODE REVIEW

File: web/.eslintrc.cjs [hunk 1]
```diff
@@ -8,6 +8,14 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
   plugins: ['react-refresh'],
+  overrides: [
+    {
+      files: ['*.cjs'],
```
Comment (0916dhkim) [inline web/.eslintrc.cjs:13]: #### Would Fix
I think it would be more specific if we add 'postcss.config.cjs' in place of '*.cjs'.

**Review by 0916dhkim — APPROVED** [review]
Looks good overall :+1: 
Just one minor suggestion about the eslint config.

File: web/.eslintrc.cjs (hunk 1 above)
Comment (igMike-V) [reply web/.eslintrc.cjs:13]: Done. Needed to add .eslintrc.cjs since is also complaining about module.exports

**Comment by 0916dhkim** [comment]
Looks good to me. Let's get it merged :rocket:

**Pull request merged**